test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -780,6 +780,23 @@ Describe "Invoke-WebRequest tests" -Tags "Feature", "RequireAdminOnWindows" {
         $result.Output.RelationLink["next"] | Should -BeExactly "${baseUri}?maxlinks=3&linknumber=2&type=${type}"
     }
 
+    It "Verify Invoke-WebRequest supresses terminating errors with -SkipHttpErrorCheck" {
+        $Query = @{
+            statuscode = 404
+            responsephrase = 'Not found'
+            contenttype = 'text/plain'
+            body = 'oops'
+            headers = "{}"
+        }
+
+        $Uri =  Get-WebListenerUrl -Test 'Response' -Query $Query
+        $command = "Invoke-WebRequest -SkipHttpErrorCheck -Uri '$uri'"
```

Comment:
I'd expect a check if we do not have SkipHttpErrorCheck parameter to ensure that the parameter really works.

Reply:
there are already tests for error handling that would have failed if ` -SkipHttpErrorCheck` was present. I think those are sufficient to test for and prevent regression.

Reply:
are all the relevant tests in https://github.com/PowerShell/PowerShell/blob/master/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1 (branch:`master`) ?

Reply:
yes

Reply:
If there are tests for errors they is other test - they could be changed or removed. My request here is make the test more clear and reliable - that for the _same_ input/environment we have predictable behavior with and without the parameter.

Reply:
I agree with iSazonov. The other tests could be changed for whatever reason related to what they're meant to test. I've added failure tests.